src/services/codefixes/importFixes.ts

```diff
@@ -322,7 +322,7 @@ namespace ts.codefix {
             tryGetModuleNameAsNodeModule(options, moduleFileName, host, getCanonicalFileName, sourceDirectory) ||
             tryGetModuleNameFromBaseUrl(options, moduleFileName, getCanonicalFileName) ||
             options.rootDirs && tryGetModuleNameFromRootDirs(options.rootDirs, moduleFileName, sourceDirectory, getCanonicalFileName) ||
-            removeFileExtension(getRelativePath(moduleFileName, sourceDirectory, getCanonicalFileName));
+            removeExtensionAndIndexPostFix(getRelativePath(moduleFileName, sourceDirectory, getCanonicalFileName));

```

Comment:
You shouldnt be removing file ```index``` when module resolution is not node_modules right? Because classicModuleResolver doesnt auto lookup for file with name of index in moduleName directory?